src/Components/GraduatesList/GraduatesList.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+import { Checkbox, Input, Table } from "semantic-ui-react";
+import FilterButton from "./FilterButton";
+import Graduated from "./Graduated";
+import axios from "axios";
+
+class GraduatesList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      graduates: []
+    };
+  }
+
+  componentDidMount() {
+    const API_URL = process.env.REACT_APP_API_URL;
+    axios
+      .get(`${API_URL}/empresas-egresades`)
+      .then(response => {
+        this.setState({
+          graduates: response.data
+        });
+      })
+      .catch(function(error) {
+        console.log(error);
```

Comment:
I dont know if console.log needs to be included on develop branch, i could be a critical bug.

Reply:
Can you give me examples of possible cases that this line causes a bug, please? Thanks.

Reply:
Yes, for example a console.log in production could expose the API, Something like "Status: 404 from https: // NahualBackend /api/getEgresades" or something like that, we have to use console.log only in development phase to check what happens with some problems. :D